earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp

```diff
@@ -655,7 +659,7 @@ void RasterAssetWidget::AssembleEditRequest(
                           + filename + "\n" +
                           khErrnoException::errorString(errno));
       case SourceConfig::NonVolume:
-        throw khException(filename + tr(" doesn't reside on a known volume."));
+        throw khException(filename + tr(" doesn't reside on a known volume. \n Make sure the volume is configured correctly using geconfigureassetroot and try again."));
```

Comment:
@tst-ppenev : Hint to use geconfigureassetroot is on purpose, it should help determine the volume ( using --listvolume) and edit or add volume if needed. Move option( as you proposed) is pretty much specific action resolution that might not work for all errors.

If this does not seem reasonable, please propose one the error message.